ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -871,6 +871,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: e2e-gcp-ipi-ovn-mtu-migrate-p2
+  cron: 0 22 */7 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@gcp-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.12'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-ovn-mtu-migrate    
```

Comment:
Extra white spaces at the end of this line cause `ci/prow/ci-operator-config-metadata`  failure.

Reply:
Thanks @liangxia fixed it. That was bad :( Still trying to fix editor settings on my new refreshed laptop